src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -118,6 +121,7 @@ def test_dmg_storage_query_pools(self):
         no_pool_info = self.get_pool_info()
         self.assertFalse(no_pool_info, "No pools should be detected.")
 
+    @skipForTicket("DAOS-4286")
```

Comment:
DAOS-4286 is **Resolved**.  You can't skip a test for a ticket that has been resolved.